### PR TITLE
Use https for link on markdown doc

### PR DESCRIPTION
**Pull request by eatjeff**

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
@eatjeff
Pls sign the CLA in order for the PR to be reviewed.
more info
https://github.com/kubernetes/community/blob/master/CLA.md

**Review by onlydole — APPROVED** [review]
Thank you, @eatjeff! 

/lgtm

**Comment by onlydole** [comment]
All that's left is the issue with the CNCF-CLA, and then this can be merged in, @eatjeff

File: content/en/blog/_posts/2017-05-00-Kubernetes-Monitoring-Guide.md [hunk 1]
```diff
@@ -79,6 +79,6 @@ _--Jean-Mathieu Saponaro, Research & Analytics Engineer, Datadog_
 
 
 - Get involved with the Kubernetes project on [GitHub](https://github.com/kubernetes/kubernetes)&nbsp;
-- Post questions (or answer questions) on [Stack Overflow](http://stackoverflow.com/questions/tagged/kubernetes)&nbsp;
+- Post questions (or answer questions) on [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes)&nbsp;
```
Comment (kbhawkey) [inline content/en/blog/_posts/2017-05-00-Kubernetes-Monitoring-Guide.md:82]: Is this a duplicate of PR https://github.com/kubernetes/website/pull/26304?

**Comment by kbhawkey** [comment]
👋 @eatjeff .
Would you verify that you signed the CLA? Thanks!
/cc @kubernetes/sig-docs-id-reviews

**Comment by kbhawkey** [comment]
Related to PR https://github.com/kubernetes/website/pull/26305.

**Comment by habibrosyad** [comment]
Thanks @eatjeff
/lgtm

**Comment by mrbobbytables** [comment]
The CLA has not been signed, going to go ahead and close this for now. It can be reopened at a later date after the CLA issue has been resolved.

/close

**Pull request closed without merging**